src/components/ContractWritePanel.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <app-panel>
+    <template v-if="aciWriteEntrypoints">
+      <h3 class="contract-write-panel__title">
+        Write Smart Contract Information
+      </h3>
+      <div class="contract-write-panel__container">
+        <wallet-account-controls
+          :backlink="route.fullPath"
+          :has-menu="false"/>
+        <hint-tooltip>
+          {{ contractVerifiedHints.connectWallet }}
+        </hint-tooltip>
+      </div>
+      <contract-entrypoint-accordion
+        :is-disabled="isDisabled"
+        :entrypoints="aciWriteEntrypoints"
+        :loading-index="loadingIndex"
+        :response="response"
+        @query-clicked="getEntrypointResponse"/>
+    </template>
+    <blank-state v-else/>
+  </app-panel>
+</template>
+
+<script setup>
+import { useContractVerifiedStore } from '@/stores/contractVerified'
+import { useContractDetailsStore } from '@/stores/contractDetails'
+import { contractVerifiedHints } from '@/utils/hints/contractVerifiedHints'
+
+const contractVerifiedStore = useContractVerifiedStore()
+const contractDetailStore = useContractDetailsStore()
+const { aciWriteEntrypoints } = storeToRefs(contractVerifiedStore)
+const { contractDetails } = storeToRefs(contractDetailStore)
+
+const {
+  fetchEntrypointResponse,
+  parseArguments,
+  parseResponse,
+  getWriteContractInstance,
+  walletSdk,
+} = contractVerifiedStore
+
+const route = useRoute()
+
+const response = ref([])
+const loadingIndex = ref(null)
+
+const isDisabled = computed(() => {
+  const connectedAddress = walletSdk ? Object.keys(walletSdk._accounts.current)[0] : null
+  return !walletSdk || contractDetails.value.createdBy !== connectedAddress
```

Comment:
```suggestion
  return !walletSdk
```
This is not an obvious restriction, contract should manage permissions by itself

Reply:
right, fixed